redhat-operators/kie-cloud-operator/kiecloud-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kiecloud-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: The operator's certification. This field should be set to false until a process to get this value is defined at a later date.
+    description: KieCloud can deploy RHPAM environments in the form of KieApp objects.
+    containerImage: quay.io/kiegroup/kie-cloud-operator:v0.1.0
+    createdAt: The date that the operator was created. The format should match yyyy-mm-ddThh:mm:ssZ
```

Comment:
This should be set to the operator creation date appropriately.

Reply:
done